src/traceViewer/videoTileGenerator.ts

```diff
@@ -30,13 +30,24 @@ export class VideoTileGenerator {
   }
 
   async render(events: PageVideoTraceEvent[]) {
+    let ffmpegName = '';
+    if (process.platform === 'win32') {
+      if (process.arch === 'ia32')
+        ffmpegName = 'ffmpeg-win32';
+      else
+        ffmpegName = 'ffmpeg-win64';
+    }
+    if (process.platform === 'darwin')
+      ffmpegName = 'ffmpeg-mac';
+    if (process.platform === 'linux')
+      ffmpegName = 'ffmpeg-linux';
+    const ffmpeg = path.join(path.dirname(require.resolve('playwright')), 'third_party', 'ffmpeg', ffmpegName);
```

Comment:
this logic looks very similar to a logic which we have upstream. Maybe we can expose it somehow internally as a util function?

Reply:
i don't think it is worth it